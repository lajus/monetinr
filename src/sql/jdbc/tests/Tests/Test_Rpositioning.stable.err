stderr of test 'Test_Rpositioning` in directory 'sql/jdbc/tests` itself:


# 22:23:25 >  
# 22:23:25 >  Mtimeout -timeout 180 Mserver "--config=/nfs/poseidon/Temp/monet/program/etc/monet.conf" --debug=10 --set "monet_mod_path=/nfs/poseidon/Temp/monet/program/lib/MonetDB" --set "gdk_dbfarm=/nfs/poseidon/Temp/monet/program/var/MonetDB/dbfarm" --set "sql_logdir=/nfs/poseidon/Temp/monet/program/var/MonetDB/log" --set mapi_port=45030 --set sql_port=52396 --set monet_prompt= --trace "--dbname=mTests_src_jdbc_tests" "/nfs/poseidon/Temp/monet/build-head/sql/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 22:23:25 >  


# 22:23:25 >  
# 22:23:25 >  Mtimeout -timeout 60 ./Test_Rpositioning.SQL Test_Rpositioning 
# 22:23:25 >  


# 22:23:25 >  
# 22:23:25 >  java -classpath /nfs/poseidon/Temp/monet/program/share/sql/Tests:/nfs/poseidon/Temp/monet/program/share/MonetDB/lib/MonetJDBC.jar Test_Rpositioning jdbc:monetdb://poseidon:52396/database?user=monetdb&password=monetdb
# 22:23:25 >  


# 22:23:26 >  
# 22:23:26 >  Done.
# 22:23:26 >  

