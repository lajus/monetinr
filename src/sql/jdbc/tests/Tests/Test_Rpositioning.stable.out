stdout of test 'Test_Rpositioning` in directory 'sql/jdbc/tests` itself:


# 22:23:25 >  
# 22:23:25 >  Mtimeout -timeout 180 Mserver "--config=/nfs/poseidon/Temp/monet/program/etc/monet.conf" --debug=10 --set "monet_mod_path=/nfs/poseidon/Temp/monet/program/lib/MonetDB" --set "gdk_dbfarm=/nfs/poseidon/Temp/monet/program/var/MonetDB/dbfarm" --set "sql_logdir=/nfs/poseidon/Temp/monet/program/var/MonetDB/log" --set mapi_port=45030 --set sql_port=52396 --set monet_prompt= --trace "--dbname=mTests_src_jdbc_tests" "/nfs/poseidon/Temp/monet/build-head/sql/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 22:23:25 >  

# Monet Database Server V4.3.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.


Ready.


# 22:23:25 >  
# 22:23:25 >  Mtimeout -timeout 60 ./Test_Rpositioning.SQL Test_Rpositioning 
# 22:23:25 >  


# 22:23:25 >  
# 22:23:25 >  java -classpath /nfs/poseidon/Temp/monet/program/share/sql/Tests:/nfs/poseidon/Temp/monet/program/share/MonetDB/lib/MonetJDBC.jar Test_Rpositioning jdbc:monetdb://poseidon:52396/database?user=monetdb&password=monetdb
# 22:23:25 >  

1. true	true
2. false	false
3. true	true
4. false	false
5. true	true
6. false	false
7. true	true
8. false	false
9. true	true
10. false	false
11. false	false
12.true	true
1. true	true
2. false	false
3. true	true
4. false	false
5. true	true
6. false	false
7. true	true
8. false	false
9. true	true
10. false	false
11. false	false
12. true	true

# 09:09:12 >  
# 09:09:12 >  Done.
# 09:09:12 >  

